### PR TITLE
feat(google): adds ability to package deck/google as a library

### PR DESCRIPTION
**Pull request by danielpeach**
@anotherchrisberry would appreciate a quick look

### CODE REVIEW

File: app/scripts/modules/google/src/loadBalancer/configure/http/createHttpLoadBalancer.controller.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = angular.module('spinnaker.deck.gce.loadBalancer.createHttp.cont
   require('./backendService/backendService.component.js'),
   require('./basicSettings/basicSettings.component.js'),
   GCE_CACHE_REFRESH,
-  require('core/modal/wizard/wizardSubFormValidation.service.js'),
+  require('@spinnaker/core/modal/wizard/wizardSubFormValidation.service.js'),
```
Comment (anotherchrisberry) [inline app/scripts/modules/google/src/loadBalancer/configure/http/createHttpLoadBalancer.controller.js]: You might need to convert this file to TS and export it in the global namespace (not now, necessarily, but before we split everything into its own repository).

File: app/scripts/modules/google/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    umdNamedDefine: true,
+  },
+  externals: {
+    '@spinnaker/core': '@spinnaker/core',
+    'angular': 'angular',
+    'lodash': 'lodash',
+    'rxjs': 'rxjs',
+    '@uirouter/core': '@uirouter/core',
+    '@uirouter/angularjs': '@uirouter/angularjs',
+    'exports-loader?"n3-line-chart"!n3-charts/build/LineChart.js': 'exports-loader?"n3-line-chart"!n3-charts/build/LineChart.js',
```
Comment (anotherchrisberry) [inline app/scripts/modules/google/webpack.config.js]: You don't need this one, since the google module doesn't use the n3-line-chart library.

**Comment by anotherchrisberry** [comment]
A few very little things, but nothing that needs to be fixed in order to merge this - nice work!

File: app/scripts/modules/google/webpack.config.js (hunk 2 above)
Comment (danielpeach) [inline app/scripts/modules/google/webpack.config.js]: Good point, thanks.

**Pull request merged**